event_routing_backends/management/commands/transform_tracking_logs.py

```diff
@@ -35,8 +35,8 @@ def _get_chunks(source, file, start_byte, end_byte):
         try:
             chunks = source.download_object_range_as_stream(
                 file,
-                start_bytes=start_byte,
-                end_bytes=end_byte
+                start_bytes=0,
+                chunk_size=CHUNK_SIZE
```

Comment:
I feel like I tried this at some point and it didn't work as I expected it to. Have you made sure that you're getting chunks of the expected sizes?

Reply:
No I didn't, and it didn't solve the problem.